core/test/Node.js

```diff
@@ -236,6 +236,19 @@ test('Node', function(t) {
         t.end();
     });
 
+    t.test('getPositionFrom method', function(t) {
+        var root  = (new Node())    .setPosition(0,  0,  0);
+        var node1 = root.addChild() .setPosition(10, 10, 10);
```

Comment:
The chosen values could be more useful. E.g. since x, y, z are all the same, the test doesn't test if `getPositionFrom` actually determines the difference along the correct axis (on a certain node).